apps/projection/src/app/ui/card/card.component.ts

```diff
@@ -21,14 +26,17 @@ export class CardComponent {
 
   constructor(
     private teacherStore: TeacherStore,
-    private studentStore: StudentStore
+    private studentStore: StudentStore,
+    private cityStore: CityStore
```

Comment:
The goal of this exercice is to create a highly customizable component. No logic should be inside this component. 

you can look at ng-content and templateOutlet for projecting content